src/field_theory/ratfunc.lean

```diff
@@ -306,6 +310,18 @@ instance [is_domain K] : nontrivial (ratfunc K) :=
 ⟨⟨0, 1, mt (congr_arg to_fraction_ring) $
   by simpa only [← of_fraction_ring_zero, ← of_fraction_ring_one] using zero_ne_one⟩⟩
 
+/-- `ratfunc K` is isomorphic to the field of fractions of `polynomial K`, as rings.
+
+This is an auxiliary definition; `simp`-normal form is `is_localization.alg_equiv`.
+-/
+@[simps apply] def to_fraction_ring_ring_equiv : ratfunc K ≃+* fraction_ring (polynomial K) :=
```

Comment:
What's the reason for not making this an alg_equiv over `polynomial K`?

Reply:
This is the `aux_equiv` that was in the file originally, just `symm` of it. At this position in the file, `semiring (ratfunc K)` is not yet known. I can refactor the `field` instance later to make it available.

Reply:
But even that doesn't build the `algebra`  instance yet.

Reply:
Oh, the sneaky thing here is that `≃+*` only needs `has_mul` and `has_add`, not a full (nonassoc) semiring instance, right?

Reply:
Precisely